### PR TITLE
Added code in python to find the Inverse of array

### PR DESCRIPTION
**Pull request by Ritikakatoch0128**
Hey @rudrakshi99 @raksha009 @anjul2000 
I have made this PR to find the inverse of the array in python. It includes all the details.
Kindly review it and merge it.
Thankyou

### CODE REVIEW

**Comment by Ritikakatoch0128** [comment]
@rudrakshi99 review my PR and let me know if any changes are required.

**Review by raksha009 — COMMENTED** [review]
Plz link yr code to the appropriate file. Plz go through the guidelines and the pinned issues.

**Comment by Ritikakatoch0128** [comment]
> Plz link yr code to the appropriate file. Plz go through the guidelines and the pinned issues.

Hey I have updated the readme file .kindly review and merge

File: Code/Python/inverseArray.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+If the array elements are swapped with their correseponding
+indices,we get the inverse of an array
+
+For Exaample:
+Let arr be an array such as:
+ arr=[2,4,1,0,3]
+ In this array,at index[0]-2,index[1]-4,index[2]-1 and so on.
+
+ Now,to find inverse of array ,swap the index and the value 
+ of array
```
Comment (raksha009) [inline Code/Python/inverseArray.py:11]: Combine line 10 and 11

File: Code/Python/inverseArray.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+ After swaping ,arr=[3,2,0,4,1]
+
+ Note:The elements must be unique and less than the length of array
+
```
Comment (raksha009) [inline Code/Python/inverseArray.py:15]: Remove line 16 to 36

File: Code/Python/inverseArray.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+4
+1 3 0 2
+
+Sample Output:
+3 2 0 4 1
+2 0 3 1
+
+Time complexity: O(n)
+'''
+def inputElements(n):
```
Comment (raksha009) [inline Code/Python/inverseArray.py:38]: Add single line comments above every func stating what its is doing

File: Code/Python/inverseArray.py [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+    arr=[]
+    for i in range(n):
+         ele=int(input())
```
Comment (raksha009) [inline Code/Python/inverseArray.py:41]: Before taking an input, tell what to input as in no of elements the user has to enter and what does that entered no denote.

File: Code/Python/inverseArray.py [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+         arr.append(ele)
```
Comment (raksha009) [inline Code/Python/inverseArray.py:42]: Use appropriate variable names

File: Code/Python/inverseArray.py [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+    return lst1
+#main code
+t=int(input()) 
+for i in range(0,t):
+    l=int(input())
+    arr=inputElements(l)
+    arr2=inverseArray(arr)
+    print(arr2)
+
+
```
Comment (raksha009) [inline Code/Python/inverseArray.py:60]: Plz read pinned issues. and make changes accordingly here too

**Review by raksha009 — CHANGES_REQUESTED** [review]

**Comment by raksha009** [comment]
Resolve conflicts too.

File: Code/Python/inverseArray.py (hunk 6 above)
Comment (Ritikakatoch0128) [reply Code/Python/inverseArray.py:39]: > Plz read pinned issues. and make changes accordingly here too

@raksha009 You can review the PR  now I have made all necessary changes .

File: Code/Python/inverseArray.py [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+#inserts the elements in an array of size n
+def inputElements(n):
+    arr=[]
+    for i in range(n):
+        #enter element to insert in array
+         element=int(input())
+         arr.append(element)
+    print("original array:",arr)
+    return arr
+#function to find inverse of array
```
Comment (raksha009) [inline Code/Python/inverseArray.py:15]: Add a new line above line 15

File: Code/Python/inverseArray.py [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+t=int(input()) 
+for i in range(0,t):
+    # enter length of array
+    array_length=int(input())
+    arr=inputElements(array_length)
+    arr2=inverseArray(arr)
+    print("Inverse of array :",arr2)
+
+'''
+TestCases :
```
Comment (raksha009) [inline Code/Python/inverseArray.py:34]: Hey, do not accept test cases. Have a look at few merged prs.

Comment (raksha009) [reply Code/Python/inverseArray.py:34]: You directly need to write 
Test Cases
Input :
Output :

**Review by raksha009 — CHANGES_REQUESTED** [review]
Do these few changes, we'll merge yr pr then

File: Code/Python/inverseArray.py (hunk 8 above)
Comment (Ritikakatoch0128) [reply Code/Python/inverseArray.py:33]: > Hey, do not accept test cases. Have a look at few merged prs.

@raksha009 please review now, I have made the suggested changes

**Review by raksha009 — APPROVED** [review]

**Comment by raksha009** [comment]
Hey issue no ?

**Comment by raksha009** [comment]
Heyy, issue no ?

**Comment by Ritikakatoch0128** [comment]
> Heyy, issue no ?
> 

#847

**Pull request merged**